azerothcore/azerothcore-wotlk
# fix(DB/quest): Recover the cargo! objects not despawning

## Changes Proposed:
-  Despawn gameobject when the item given in the quest Salvage Kit (https://wowgaming.altervista.org/aowow/?item=33044) is used on Shipwreck Debris (https://wowgaming.altervista.org/aowow/?object=186283), for the quest "Recover the Cargo!" (https://wowgaming.altervista.org/aowow/?quest=11140)

## Issues Addressed:

- Closes #5632
- Closes https://github.com/chromiecraft/chromiecraft/issues/523

## SOURCE:


## Tests Performed:

- Object despawns in my dev environment after applying the fix: https://streamable.com/9y28x1


## How to Test the Changes:


1. `.go c id 23569`
2. `.q a 11140`
3. Use object given by the quest and check that Shipwreck Debris despawns

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
